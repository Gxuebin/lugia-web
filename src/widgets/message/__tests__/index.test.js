/**
 *
 * create by guorg
 *
 * @flow
 */
import React from 'react';
import chai from 'chai';
import 'jest-styled-components';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Message from '../demo';

const { expect: exp } = chai;

Enzyme.configure({ adapter: new Adapter() });

describe('Message', () => {
  it('css', () => {
    const Target = <Message />;
    expect(renderer.create(Target).toJSON()).toMatchSnapshot();
  });
});
