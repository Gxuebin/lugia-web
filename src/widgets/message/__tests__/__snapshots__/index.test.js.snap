// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message css 1`] = `
.c0 {
  display: inline-block;
  margin-bottom: 0;
  box-sizing: border-box;
  text-align: center;
  touch-action: manipulation;
  cursor: pointer;
  white-space: nowrap;
  line-height: 1;
  font-family: Trebuchet Ms,Arial,Helvetica,sans-serif;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: all 0.3s cubic-bezier(0.645,0.045,0.355,1);
  transition: all 0.3s cubic-bezier(0.645,0.045,0.355,1);
  position: relative;
  text-transform: none;
  outline: 0;
  color: #666666;
  background-color: #ffffff;
  border: 1px solid #cccccc;
  padding: 0 1.2857142857142858em;
  font-size: 1.4rem;
  border-radius: 0.28571428571428575em;
}

.c0:hover {
  color: #8090ff;
  border: 1px solid #8090ff;
  background-color: rgba(77,99,255,0.05);
}

.c0:focus {
  color: #8090ff;
  border: 1px solid #8090ff;
  background-color: rgba(77,99,255,0.05);
}

.c0:active {
  color: #3e4fcc;
  border: 1px solid #3e4fcc;
}

.c1 {
  display: inline-block;
  line-height: 2.142857142857143em;
}

<div>
  <br />
  <br />
  <button
    className="c0"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    size="default"
  >
    <span
      className="c1"
      size="default"
    >
      普通提示
    </span>
  </button>
  <br />
  <br />
  <button
    className="c0"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    size="default"
  >
    <span
      className="c1"
      size="default"
    >
      成功提示
    </span>
  </button>
  <br />
  <br />
  <button
    className="c0"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    size="default"
  >
    <span
      className="c1"
      size="default"
    >
      错误提示
    </span>
  </button>
  <br />
  <br />
  <button
    className="c0"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    size="default"
  >
    <span
      className="c1"
      size="default"
    >
      警告提示
    </span>
  </button>
  <br />
  <br />
  <button
    className="c0"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    size="default"
  >
    <span
      className="c1"
      size="default"
    >
      加载提示
    </span>
  </button>
</div>
`;
